src/crypto/CHIPCryptoPAL.h

```diff
@@ -73,10 +73,7 @@ inline constexpr size_t kMAX_Hash_Length            = kSHA256_Hash_Length;
 // Minimum required CSR length buffer length is relatively small since it's a single
 // P256 key and no metadata/extensions are expected to be honored by the CA.
 inline constexpr size_t kMIN_CSR_Buffer_Size = 255;
-
-[[deprecated("This constant is no longer used by common code and should be replaced by kMIN_CSR_Buffer_Size. Checks that a CSR is "
-             "<= kMAX_CSR_Buffer_size must be updated. This remains to keep valid buffers working from previous public API "
-             "usage.")]] constexpr size_t kMAX_CSR_Buffer_Size = 255;
```

Comment:
You cannot change this deprecation without justification. Changing this value will break stuff.